src/google-recaptcha-provider.tsx

```diff
@@ -99,14 +102,20 @@ export class GoogleReCaptchaProvider extends React.Component<IGoogleReCaptchaPro
   };
 
   handleOnLoad = () => {
+    const { enterprise } = this.props;
+
     if (!window || !(window as any).grecaptcha) {
       console.warn(GoogleRecaptchaError.SCRIPT_NOT_AVAILABLE);
-
       return;
     }
 
-    (window as any).grecaptcha.ready(() => {
-      this.resolver((window as any).grecaptcha);
+    let _grecaptcha = (window as any).grecaptcha;
```

Comment:
I would like to avoid reassignment / mutation whenever it's possible, can we do something like:

```
const grecaptcha = enterprise ? (window as any).grecaptcha.enterprice : (window as any).grecaptcha;
grecaptcha.ready(() => {...});
```